collections-csharp/UpdateItemInListCSharp/UpdateItemInListCSharp.Test/LibraryTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using UpdateItemInListCSharp.Models;
+
+namespace UpdateItemInListCSharp.Test
+{
+    public class LibraryTest
+    {
+        private readonly Library _library = new Library();
+
+        [Fact]
+        public void WhenCheckoutBookFind_ThenBookIsCheckedOut()
+        {
+            // Arrange
+            var book = new Book("Dolor sit amet Sadipscing", "Lorem Ipsum", "123-0123456789", false);
+            _library.Books.Add(book);
+
+            // Act
+            _library.CheckoutBookUsingFind("123-0123456789");
+            var checkoutBook = _library.Books.Find(b => b.ISBN == "123-0123456789");
```

Comment:
We don't need to find an object again, the same book object will be checkout. We can directly check if book.IsCheckOut is true or not

Reply:
Okay, that's smarter. Should I remove all "var checkoutBook = _library.Books.Find(b => b.ISBN == "123-0123456789");" lines in every unit test?

Reply:
Yes, this comment is applicable for all unit test